adopt-blog.html

```diff
@@ -0,0 +1,90 @@
+
+                <p>Kilsie eitislissi zposesko cha. Abjlkseee jlkt isani stylesheet lked eialse. Lipsseis jkj eiso spiroot, lkjlke kjkeise. Kilsie eitislissi zposesko cha.</p>    
+            </div>
+            <div class="dog-blog">
+                <img src="images/blog-2.jpg" alt="Descripion of featured dog.">
+                <h2>How To Walk Multiple Dogs</h2>
+                <p>Kilsie eitislissi zposesko cha. Abjlkseee jlkt isani stylesheet lked eialse. Lipsseis jkj eiso spiroot, lkjlke kjkeise. Kilsie eitislissi zposesko cha.</p>    
+                <p>Abjlkseee jlkt isani stylesheet lked eialse. Lipsseis jkj eiso spiroot, lkjlke kjkeise. Kilsie eitislissi zposesko cha. Abjlkseee jlkt isani stylesheet lked eialse. Lipsseis jkj eiso spiroot, lkjlke kjkeise. Kilsie eitislissi zposesko cha. Abjlkseee jlkt isani stylesheet lked eialse. Lipsseis jkj eiso spiroot, lkjlke kjkeise. Kilsie eitislissi zposesko cha. Abjlkseee jlkt isani stylesheet lked eialse. Lipsseis jkj eiso spiroot, lkjlke kjkeise. Kilsie eitislissi zposesko cha.</p>    
+
+            </div>
+            <div class="dog-blog">
+                <img src="images/blog-3.jpg" alt="Descripion of featured dog.">
+                <h2>Teach Your Dog To Fetch</h2>
+                <p>Abjlkseee jlkt isani stylesheet lked eialse. Lipsseis jkj eiso spiroot, lkjlke kjkeise. Kilsie eitislissi zposesko cha. Abjlkseee jlkt isani stylesheet lked eialse. Lipsseis jkj eiso spiroot, lkjlke kjkeise. Kilsie eitislissi zposesko cha. Abjlkseee jlkt isani stylesheet lked eialse. Lipsseis jkj eiso spiroot, lkjlke kjkeise. Kilsie eitislissi zposesko cha. Abjlkseee jlkt isani stylesheet lked eialse. Lipsseis jkj eiso spiroot, lkjlke kjkeise. Kilsie eitislissi zposesko cha.</p>    
+
+                <p>Kilsie eitislissi zposesko cha. Abjlkseee jlkt isani stylesheet lked eialse. Lipsseis jkj eiso spiroot, lkjlke kjkeise. Kilsie eitislissi zposesko cha.</p>    
+            </div>
+        </div>
+        <div>
+            <footer><img class="ftr-img" src="images/adoptadog-logo.svg" alt="Adopt A Dog Logo">
+            <p>Abjlkseee jlkt isani stylesheet lked eialse. Lipsseis jkj  <br>eiso spiroot, lkjlke kjkeise. Kilsie eitislissi zposesko cha.</p>    
+                <nav>
+                    <ul>
+                        <li><a href="index.html">Home</a></li>
+                        <li><a href="adopt-dogs.html">Dogs</a></li>
+                        <li><a href="">Blog</a></li>
+                    </ul>
+                </nav>
+                <p>2018 Adoptadog, LLC | Seattle, WA</p>
```

Comment:
this text is hard to see given the contrast of grey text on blue background. verify mockup for styling